lua/Events.lua

```diff
@@ -66,6 +70,11 @@ function library.addEventHandlers(obj)
     end
   end)
 
+  -- Removes all events
+  obj.clearEvents = (function(self)

```

Comment:
Personal suggestion, but I'd call this `clearAllEvents` just to make it clearer and have less chances of mixing up with `clearEvent`

Reply:
> Thanks for the submission @leniver! As far I understood, we're tracking objects that have event handler support and making it possible to clear all of them at once? Just wondering, is there any specific use case for this?

I am using that to reload libraries and being able to have some event garbage collection.